src/ResourceManager/AzureBatch/Commands.Batch/Jobs/GetBatchJobPreparationAndReleaseTaskStatusCommand.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Batch;
+using Microsoft.Azure.Commands.Batch.Models;
+using System.Management.Automation;
+using Constants = Microsoft.Azure.Commands.Batch.Utils.Constants;
+
+namespace Microsoft.Azure.Commands.Batch
+{
+    [Cmdlet(VerbsCommon.Get, Constants.AzureBatchJobPrepAndReleaseStatus, DefaultParameterSetName = Constants.IdParameterSet),
+        OutputType(typeof(PSJobPreparationAndReleaseTaskExecutionInformation))]
+    public class GetBatchJobPreparationAndReleaseTaskStatusCommand : BatchObjectModelCmdletBase
+    {
+        [Parameter(Position = 0,
+            ParameterSetName = Constants.IdParameterSet,
+            Mandatory = true,
+            HelpMessage = "Specifies the ID of the job whose preparation and release tasks should be retrieved. You cannot specify wildcard characters.")]
+        [ValidateNotNullOrEmpty]
+        public string Id { get; set; }
+
+        [Parameter(
+            Position = 0,
+            ParameterSetName = Constants.InputObjectParameterSet,
+            ValueFromPipeline = true,
+            Mandatory = true,
+            HelpMessage = "Specifies a PSCloudJob object that represents the job to get the preparation and release task status from. To obtain a PSCloudJob object, use the Get-AzureBatchJob cmdlet.")]
+        [ValidateNotNullOrEmpty]
+        public PSCloudJob Job { get; set; }
```

Comment:
@matthchr I would rename this parameter to `InputObject` since it is the generic name used throughout PowerShell cmdlets to represent an object that can be passed through the pipeline and a pattern that we are trying to enforce more for new cmdlets. You could then use `Job` as an alias to this parameter.